qemu/tests/cfg/usb.cfg

```diff
@@ -338,6 +338,51 @@
                 - usb_multi_times:
                     no usb_negative_test
                     usb_repeat_times = 20
+        - usb_redir:
+            type = usb_redir
+            only Linux
+            no Host_RHEL.m8.u3
+            usb_devices += " usbredir1"
+            usb_type_usbredir1 = usb-redir
+            chardev_id_usbredir1 = usbredir_chardev1
+            chardev_backend_usbredir1 = spicevmc
+            chardev_name_usbredir1 = usbredir
+            usbdev_option_chardev_usbredir1 = usbredir_chardev1
+            spice_port = 5900
+            del spice_password
+            del spice_addr
+            del spice_image_compression
+            del spice_zlib_glz_wan_compression
+            del spice_streaming_video
+            del spice_agent_mouse
+            del spice_playback_compression
+            del spice_ipv4
+            # must configure which device should be used
+            #usbredir_vendorid = xxxx
+            #usbredir_productid = xxxx
+            variants option:
+                - @basic:
+                    only spice
```

Comment:
`spice` and `vnc` are internal variants,  better do not use it if we can achieve the same goal with other method ?  Does define `display = spice` match your test requirement ?

Reply:
And it can be setted with the parent variant `usb_redir`.

Reply:
Adding `display = spice` doesn't take effect.
And there is still another case planned that doesn't have this restriction.
Thanks!

Reply:
> Adding `display = spice` doesn't take effect.
> And there is still another case planned that doesn't have this restriction.
> Thanks!

ok, how many cases that will not use spice under 'usb_redir' in the future ?  If only one, we can use different setting for the case only.

Reply:
Only one, and it has no restriction `only spice`. So what do you mean with "use different setting for the case only"?

Reply:
> Only one, and it has no restriction `only spice`. So what do you mean with "use different setting for the case only"?

Ok~  if the left case do not need any restriction,  it is ok.

Reply:
Updated, thanks!